server/routes/oodler.js

```diff
@@ -22,15 +22,26 @@ function get(request, reply) {
 }
 
 function create(request, reply) {
-  return Oodler({
-    firstName: request.payload.firstName,
-    lastName: request.payload.lastName,
-    email: request.payload.email,
-    office: request.payload.office
-    })
-    .save()
-    .then(result => {
-      reply(result).code(201);
+  return Oodler
+    .filter({ email: request.payload.email})
+    .run()
+    .then(exists => {
```

Comment:
Please rename this variable to `oodlers`. While more readable as an `exists`, it's not what the `filter()` method returns. For example, if I knew that it was an array of `Oodlers`, I would maybe want to change the message to `User already registered with email ${oodlers[0].email}`. Just seeing an `exists` takes a couple of seconds of mental exercise we can avoid.

As an aside: in most ORMs the uniqueness (and any other) validation is specified on the model directly. That way this check can be skipped in this and all other instances. Always try to find a way to enforce it in that fashion.

P.S. :+1: